jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/java/com/atlassian/jira/plugins/slack/service/listener/impl/DefaultIssueEventToEventMatcherTypeConverter.java

```diff
@@ -51,11 +54,24 @@ public DefaultIssueEventToEventMatcherTypeConverter(final ChangeLogExtractor cha
             String field = change.getField();
             if (ChangeLogExtractor.ASSIGNEE_FIELD_NAME.equals(field)) {
                 eventMatcherTypes.add(EventMatcherType.ISSUE_ASSIGNMENT_CHANGED);
-            } else if (ChangeLogExtractor.STATUS_FIELD_NAME.equals(field)) {
-                eventMatcherTypes.add(EventMatcherType.ISSUE_TRANSITIONED);
             }
         }
 
         return eventMatcherTypes;
     }
+
+    @Override
+    public Collection<EventMatcherType> match(IssueChangedEvent event) {
+        Collection<ChangeItemBean> changeItems = event.getChangeItems();
```

Comment:
As a side note: I find it weird the attribute is named `changeItems`. I'd expected it to be `changedItems`, with `d`.

Reply:
I feel that the `change` is the main entity in this phrase. It represents a user action of changing an issue. `Items` are dependent entities. [I have read about this feature of English](https://www.englishclub.com/grammar/nouns-adjective.htm), when a couple of nouns go after each other, but only the last one is considered being a noun. All the preceding nouns "become" adjectives. Assuming that that rule is used here as well, there is no any error. [And it is a public Jira API class](https://docs.atlassian.com/software/jira/docs/api/8.5.12/com/atlassian/jira/event/issue/IssueChangedEvent.html#getChangeItems--), not a plugin class, so I cannot change it.